packages/monitor-v2/src/monitor-og/oSnapAutomation.ts

```diff
@@ -401,21 +403,29 @@ const submitProposals = async (
     });
     const explanation = ethersUtils.toUtf8Bytes(proposal.ipfs);
 
+    // Create potential error log for simulating/proposing.
+    const proposalErrorLog = {
+      at: "oSnapAutomation",
+      mrkdwn:
```

Comment:
+1 thank you for adding etherscan links on top of the snapshot ones. Makes this much more useful!